data-prepper-expression/src/main/java/org/opensearch/dataprepper/expression/DateTimeFormatExpressionFunction.java

```diff
@@ -0,0 +1,87 @@
+package org.opensearch.dataprepper.expression;
```

Comment:
Please add the license headers to this file:

https://github.com/opensearch-project/data-prepper/blob/main/CONTRIBUTING.md#license-headers